include/gprobe.php

```diff
@@ -7,9 +7,9 @@
 require_once('include/datetime.php');
 
 function gprobe_run(&$argv, &$argc){
-	if($argc != 2)
+	if ($argc != 2) {
```

Comment:
Thank you for finding these. :+1: Please keep in mind, that my upcoming PRs are changing similar stuff.